src/user/configure-metadata.md

```diff
@@ -4548,6 +4547,10 @@ Table: External map layer objects in the Maintenance app
 
       - XYZ tiles (can also be used for WMTS)
 
+      - Vector tiles (Vector Style)
+
```

Comment:
Add: "The map service URL should be to a JSON document that follows the [Mapbox GL Style Spec](https://docs.mapbox.com/mapbox-gl-js/style-spec/)."